test/trace.ts

```diff
@@ -66,22 +66,21 @@ let singleton: TraceWriter|null = null;
 
 traceWriter.create =
     (logger: common.Logger, config: TraceWriterSingletonConfig,
-     cb?: (err?: Error) => void):
-        TraceWriter => {
-          if (singleton) {
-            throw new Error('Trace Writer already created.');
-          }
-          singleton = new TestTraceWriter(logger, config);
-          singleton.initialize(cb || (() => {}));
-          return singleton;
-        },
-
-        traceWriter.get = (): TraceWriter => {
```

Comment:
Oops. Was this an error introduced in the previous PR?

Reply:
Yep, I used a naked comma instead of a semi-colon... not sure why tslint/clang-format didn't catch this 😛